app/helpers/pipeline_outputs_helper.rb

```diff
@@ -140,6 +140,18 @@ def taxon_name(taxid, tax_level)
     taxon["#{tax_level}_name"] if taxon
   end
 
+  def get_taxid_fasta_from_pipeline_run_combined_nt_nr(pipeline_run, taxid, tax_level)
+    nt_array = get_taxid_fasta_from_pipeline_run(pipeline_run, taxid, tax_level, 'NT').split(">")
+    nr_array = get_taxid_fasta_from_pipeline_run(pipeline_run, taxid, tax_level, 'NR').split(">")
+    combined_array = ((nt_array | nr_array) - [''])
+
+    if combined_array.empty?
+      return nil
```

Comment:
why would a taxon we know to have hits in a sample not have a fasta? please add a comment in the code

Reply:
Added comments to make it more clear. We don't actually have that guarantee about the taxon for this method (it IS guaranteed for how the bulk download uses the function, but it's possible that someone calls this method later with taxa that might not have reads).